00-HTML-CSS-basics/html/toDo.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
This file looks really good! As a general comment, keep an eye on files' names, since most servers will not render your page if they detect something weird on them (in this case the uppercase *"D"*). This could be named *to-do.html* or similar.